lib/OpenQA/WebAPI/Controller/API/V1/Iso.pm

```diff
@@ -360,19 +360,17 @@ sub schedule_iso {
 
             # create a new job with these parameters and count if successful, do not send job notifies yet
             my $job = $self->db->resultset('Jobs')->create_from_settings($settings);
+            push @jobs, $job;
 
-            if ($job) {
```

Comment:
question to reviewers: Should we keep the behaviour to silently ignore if a job can not be created or catch with eval?